jade-chrome.el

```diff
@@ -35,10 +35,33 @@
 
 (require 'jade-webkit)
 
+(defvar jade-chrome-bin "chromium")
+(defvar jade-chrome-port "9222")
+
+(defun jade-run-chrome (url)
+  "Start chromium with remote debugging enabled"
```

Comment:
You should end documentation sentences with a full stop.

If you're not already using it, I would recommand using flycheck, which is great for finding this typos (and many other linting issues & errors).